src/configuration/Options.cpp

```diff
@@ -200,6 +203,28 @@ MILPSolverBoundTighteningType Options::getMILPSolverBoundTighteningType() const
     }
 }
 
+SoISearchStrategy Options::getSoISearchStrategy() const
+{
+    String strategyString = String( _stringOptions.get
+                                    ( Options::SOI_SEARCH_STRATEGY ) );
+    if ( strategyString == "mcmc" )
+        return SoISearchStrategy::MCMC;
+    else if ( strategyString == "walksat" )
+        return SoISearchStrategy::WALKSAT;
+    else
+        return SoISearchStrategy::MCMC;
+}
+
+SoIInitializationStrategy Options::getSoIInitializationStrategy() const
+{
+    String strategyString = String( _stringOptions.get
+                                    ( Options::SOI_INITIALIZATION_STRATEGY ) );
+    if ( strategyString == "input-assignment" )
```

Comment:
Hmm... That's a funny if-then-else. Is this a place holder for later supporting "random"?

Reply:
Yeah, exactly. We might consider alternative initialization strategies in the future.